### ISSUE TITLE
Failed to Recognize File Format from Google Drive Link

### ISSUE THREAD

**Issue by RfadnjdExt** [mpv-player/mpv#12616]
### Important Information
- mpv version: 0.36
- macOS Version: 13.6
- Source of the mpv binary or bundle: Homebrew

### Reproduction steps
Execute the following command with MPV: 
```
mpv -v --ytdl-raw-options=cookies=drive.google.com_cookies.txt 'Google Drive Link'
```

### Expected behavior
The video should be played successfully in MPV without any errors.

### Actual behavior
MPV fails to recognize the file format and exits with an error message.

### Log file
https://0x0.st/H4qd.txt

**Comment by llyyr** [comment]
Can yt-dlp download this file?

**Comment by RfadnjdExt** [comment]
Yes, yt-dlp can download the file. I've included the yt-dlp logs for reference. Here's the confirmation log:
https://0x0.st/H4ap.txt
Please review the logs for details. If you encounter any issues or need further assistance, feel free to ask.

**Comment by Dudemanguy** [comment]
Should be fixed with #12505.
